### PR TITLE
Prometheus support for JHipster v6

### PR DESCRIPTION
**Pull request by PierreBesson**
-   [X] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [X] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [X] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by PierreBesson** [comment]
I removed the milestone as this should not be a blocker for V6.

File: generators/generator-constants.js [hunk 1]
```diff
@@ -43,13 +43,13 @@ const DOCKER_JHIPSTER_ZIPKIN = 'jhipster/jhipster-zipkin:v4.0.0';
 const DOCKER_TRAEFIK = 'traefik:1.7.9';
 const DOCKER_CONSUL = 'consul:1.4.2';
 const DOCKER_CONSUL_CONFIG_LOADER = 'jhipster/consul-config-loader:v0.3.0';
-const DOCKER_PROMETHEUS = 'prom/prometheus:v2.7.1';
+const DOCKER_PROMETHEUS = 'prom/prometheus:v2.7.2';
```
Comment (DanielFran) [inline generators/generator-constants.js]: Latest version is 2.8.0

File: generators/generator-constants.js [hunk 2]
```diff
@@ -43,13 +43,13 @@ const DOCKER_JHIPSTER_ZIPKIN = 'jhipster/jhipster-zipkin:v4.0.0';
...
 const DOCKER_PROMETHEUS_ALERTMANAGER = 'prom/alertmanager:v0.16.1';
-const DOCKER_GRAFANA = 'grafana/grafana:5.4.3';
+const DOCKER_GRAFANA = 'grafana/grafana:6.0.0';
```
Comment (DanielFran) [inline generators/generator-constants.js]: Latest version is 6.0.2

File: generators/kubernetes/templates/monitoring/prometheus.yml.ejs [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+      labels:
+        apps.kubernetes.io/component: controller
+        apps.kubernetes.io/name: prometheus-operator
+        apps.kubernetes.io/version: v0.29.0
+    spec:
+      containers:
+      - args:
+        - --kubelet-service=kube-system/kubelet
+        - --logtostderr=true
+        - --config-reloader-image=quay.io/coreos/configmap-reload:v0.0.1
```
Comment (DanielFran) [inline generators/kubernetes/templates/monitoring/prometheus.yml.ejs]: Latest version is 0.2.2 from https://hub.docker.com/r/jimmidyson/configmap-reload/tags

Comment (PierreBesson) [reply generators/kubernetes/templates/monitoring/prometheus.yml.ejs]: I'm not touching this file in the latest version of my PR, so I'm resolving this.

**Comment by PierreBesson** [comment]
This was validated by @clement26695 yesterday. Let's merge.

**Pull request merged**